orc8r/cloud/helm/orc8r/Chart.yaml

```diff
@@ -13,7 +13,7 @@ apiVersion: v1
 appVersion: "1.0"
 description: A Helm chart for magma orchestrator
 name: orc8r
-version: 1.5.2
```

Comment:
1.5.4 already exists, but due to the order the changes were landed, got overwritten